ArchiSteamFarm/Json/Steam.cs

```diff
@@ -445,6 +450,7 @@ internal sealed class Description {
 				internal ulong ClassID { get; private set; }
 				internal bool Marketable { get; private set; }
 				internal bool Tradable { get; private set; }
+				internal int Droprate { get; private set; }
```

Comment:
`int` as droprate is unacceptable, it should be reverse-engineered `Enum` that maps to `Common`, `Uncommon` and `Rare` at the minimum, perhaps more needed. Since Steam reserved `0` value, it should be a nullable type to distinguish unknown rarity.

It should also be named rarity, as it's stated by its internal name.

Reply:
It's in the category `"category": "droprate",`, so I'm sure steam calls it droprate, not rarity. I didn't just invented it's name.
And clearly steam has `droprate_0` as internal name, are you sure 0 value is reserved and can be used as unknown droprate?